src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -4259,6 +4259,40 @@ FCVT_SAFE(fcvt_l_d, d);
 
 #undef FCVT_SAFE
 
+void MacroAssembler::java_round_float(Register dst, FloatRegister src, FloatRegister ftmp) {
+  Label done;
+  li(t0, jint_cast(0.5f));
```

Comment:
Nit: Can you change this `li` into `mv`? That will be consistent with other places where we move an immediate.

Reply:
Fixed. Thanks!